consensus/src/quorum_store/network_listener.rs

```diff
@@ -50,31 +50,33 @@ impl NetworkListener {
                             .expect("Failed to send shutdown ack to QuorumStore");
                         break;
                     },
-                    VerifiedEvent::SignedBatchInfo(signed_batch_info) => {
-                        let cmd = ProofCoordinatorCommand::AppendSignature(*signed_batch_info);
+                    VerifiedEvent::SignedBatchInfo(signed_batch_infos) => {
+                        let cmd = ProofCoordinatorCommand::AppendSignature(*signed_batch_infos);
                         self.proof_coordinator_tx
                             .send(cmd)
                             .await
                             .expect("Could not send signed_batch_info to proof_coordinator");
                     },
                     VerifiedEvent::BatchMsg(batch_msg) => {
-                        let batch = batch_msg.unpack();
+                        let author = batch_msg.author();
+                        let batches = batch_msg.take();
```

Comment:
do we want to limit how many of them?

Reply:
I think this is the last remaining comment that I haven't been able to address. I'll think through if there are other cases too and push a commit tomorrow.

Reply:
Addressed in cb8e3c674d26fcf2b8730ad9ded73bd2cfa470fd